src/main/java/com/boggle/BoggleBoardGenerator.java

```diff
@@ -38,36 +38,40 @@
     }
 
     public static void main(String[] args) throws Exception {
-        ImageSize newImageSize = calculateImageSize("/Users/natashasinghvi/Documents/boggle/src/main/java/com/boggle/TASHUPhotoboggle.jpeg");
+        ImageSize newImageSize = calculateImageSize("/Users/natashasinghvi/Documents/boggle/src/main/resources/TASHUPhotoboggle.jpeg");
         int width = newImageSize.getWidth();
         int height = newImageSize.getHeight();
+        Character[][] arrayLetter = new Character[5][5];
 
         List<EntityAnnotation> finalAnnotation = new ArrayList<EntityAnnotation>();
-        finalAnnotation = returnAnnotationsViaGoogle();
+        finalAnnotation = returnAnnotationsViaGoogle("/Users/natashasinghvi/Documents/boggle/src/main/resources/TASHUPhotoboggle.jpeg");
 
         for (int i = 1; i < finalAnnotation.size(); i++){
             BoggleBoardGenerator boggleBoardGenerator = new BoggleBoardGenerator();
             BoggleAnnotations boggleAnnotations = new BoggleAnnotations(finalAnnotation.get(i), width, height, finalAnnotation.get(i).getDescription());
-            System.out.println("Index: " + i); //prints out the information for each index of finalAnnotation
             int colMin = boggleAnnotations.findClosestLineNumber(boggleAnnotations.getTopLeftX(), boggleAnnotations.rowPixelIncrement);
             int colMax = boggleAnnotations.findClosestLineNumber(boggleAnnotations.getTopRightX(), boggleAnnotations.rowPixelIncrement);
             int rowMin = boggleAnnotations.findClosestLineNumber(boggleAnnotations.getTopRightY(), boggleAnnotations.colPixelIncrement);
             int rowMax = boggleAnnotations.findClosestLineNumber(boggleAnnotations.getBottomRightY(), boggleAnnotations.colPixelIncrement);
             List<Integer> arrayRow = boggleBoardGenerator.printAllInBetween(rowMin, rowMax);
             List<Integer> arrayCol = boggleBoardGenerator.printAllInBetween(colMin, colMax);
             boggleBoardGenerator.AllVertexCombinations(arrayRow, arrayCol, boggleAnnotations.description);
-            System.out.println(map);
+        }
+        CheckVertexRepetitionTest newVertex = new CheckVertexRepetitionTest(map);
```

Comment:
why are you calling the Test here? you shouldn't be calling any class that ends with Test

Reply:
i forgot to delete this line of code - it's not supposed to be here.